### DISCUSSION TITLE
May I use GitHub mascots for a presentation?

### DISCUSSION THREAD

**Discussion by darsaveli** [community/community#22436] [New to GitHub]
<p>Hi,</p>
<p>Is it ok if I use GitHub mascots on a presentation that I’m creating for recruiters to teach them about sourcing on GitHub?<br>
Also, what type of reference will be needed? If I’m adding the names of the mascots on the page where the mascot is, would that be ok?</p>
<p>I understood from the Github documentation that using mascots in a blog where you talk about GitHub is possible, would that be in such presentation/guidebook format as well? I would be very much grateful If you confirm so I will be calm on that matter.</p>

**Comment by ossanna16** [comment, marked as answer]
<p>Hi there! Thank you for your question! Yes, in general it should be ok to use Mona, our mascot, in your presentation. I suggest you mention the source where you got the graphics from (website URL) and mention that you are not the owner or creator of the graphic.</p>

**Comment by darsaveli** [comment]
<p>Cool. I will put the source and give proper credit. Thank you for your help!</p>

**Comment by tuves** [comment]
<p>Hi <a class="mention" href="/darsaveli">@darsaveli</a>, feel free to check our assets <a href="https://github.com/logos">here</a> as well as the dos and don’ts of our logo usage :smiley:</p>

**Comment by darsaveli** [comment]
<p>I have read that before, just wanted to make sure if I understand it right and ossanna16 confirmed, but thank you for trying to help me too with this question.</p>

**Comment by SebasAnasco1517** [comment]
Hi, may I use the github logo in a presentation? It's just to show our colleagues that we use github for our projects!
